UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -70,9 +70,9 @@ private static CreateBatchResponseModel GetCreateBatchResponse()
             return new CreateBatchResponseModel()
             {
                 BatchId = batchId,
-                BatchStatusUri = $"http://tempuri.org/batch/{batchId}/status",
-                ExchangeSetBatchDetailsUri = $"http://tempuri.org/batch/{batchId}",
-                ExchangeSetFileUri = $"http://tempuri.org/batch/{batchId}/files/",
+                BatchStatusUri = $"http://filetempuri.org/batch/{batchId}/status",
```

Comment:
I would prefer that the mocked API response contained the base URL (as it would do in reality). Using the public URL in this mocked response makes it impossible to verify the URLs are correctly swapped in `FileShareService.CreateBatch`
Please revert the `CreateBatchResponseModel` model to use base URL as it was before your latest commit.

Then please change the assertions in the `WhenFSSClientReturns201_ThenCreateBatchReturns201AndDataInResponse()` test to

```
Assert.AreEqual(HttpStatusCode.Created, response.ResponseCode, $"Expected {HttpStatusCode.Created} got {response.ResponseCode}");
Assert.AreEqual(createBatchResponse.BatchId, response.ResponseBody.BatchId);

//assert the mocked API response returned to CreateBatch contains the internal BaseUrl
Assert.IsTrue(createBatchResponse.BatchStatusUri.Contains(fakeFileShareConfig.Value.BaseUrl));
Assert.IsTrue(createBatchResponse.ExchangeSetBatchDetailsUri.Contains(fakeFileShareConfig.Value.BaseUrl));
Assert.IsTrue(createBatchResponse.ExchangeSetFileUri.Contains(fakeFileShareConfig.Value.BaseUrl));

//assert FileShareService.CreateBatch() is correctly replacing the internal BaseUrl with PublicUrl
Assert.AreEqual(createBatchResponse.BatchStatusUri.Replace(fakeFileShareConfig.Value.BaseUrl, fakeFileShareConfig.Value.PublicBaseUrl), response.ResponseBody.BatchStatusUri);
Assert.AreEqual(createBatchResponse.ExchangeSetBatchDetailsUri.Replace(fakeFileShareConfig.Value.BaseUrl, fakeFileShareConfig.Value.PublicBaseUrl), response.ResponseBody.ExchangeSetBatchDetailsUri);
Assert.AreEqual(createBatchResponse.ExchangeSetFileUri.Replace(fakeFileShareConfig.Value.BaseUrl, fakeFileShareConfig.Value.PublicBaseUrl), response.ResponseBody.ExchangeSetFileUri);
```